UKHomeOffice/sgmr-service
# NMSW-1397 Remove role from table ahead of a larger fix

## AC

<table> has been used for listing voyages, people information, pleasure craft information and possibly other information.

The current code fails the new accessibility guidelines for tables because they defined 'role=x'

----

## To test

- run linting all should pass
- run tests all should pass

----

## Notes

NMSW-1397 remains in backlog with an AC to replace <table> entirely as it's not the correct element for the content we're displaying in them
